src/lib/get-marked-with-toc.ts

```diff
@@ -0,0 +1,32 @@
+import marked from 'marked';
+import { Config } from './config';
+
+// eslint-disable-next-line import/no-mutable-exports
+export let toc: TableOfContent[] = [];
+export const getHeadingRenderer = (config: Config) => {
+	toc = [];
+	const options = config.marked_options;
+	const renderer = options.renderer ?? new marked.Renderer();
```

Comment:
Same thing as previous comment, we don't want to extend the user's custom renderer.